### PR TITLE
[MOSIP-12124] converted stages to lib

### PR DESCRIPTION
**Pull request by manojsp12**

### CODE REVIEW

File: registration-processor/post-processor/registration-processor-printing-stage/pom.xml [hunk 1]
```diff
@@ -109,6 +109,11 @@
 			<artifactId>registration-processor-info-storage-service</artifactId>
 			<version>${packet.info.storage.service.version}</version>
 		</dependency>
+		<dependency>
```
Comment (loganathan-sekaran) [inline registration-processor/post-processor/registration-processor-printing-stage/pom.xml:112]: Do we need to add this Dependency?

File: registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/config/OSIConfigBean.java [hunk 2]
```diff
@@ -13,30 +13,11 @@
...
-import io.mosip.registration.processor.stages.osivalidator.OSIValidator;
-import io.mosip.registration.processor.stages.osivalidator.OSIValidatorStage;
-import io.mosip.registration.processor.stages.osivalidator.UMCValidator;
 import io.mosip.registration.processor.stages.osivalidator.utils.OSIUtils;
 
 @Configuration
 public class OSIConfigBean {
 
-	@Bean
-	public OSIValidator getOSIValidator() {
```
Comment (loganathan-sekaran) [inline registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/config/OSIConfigBean.java:26]: Do we need to remove these non stage beans ?

File: registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidatorStage.java [hunk 3]
```diff
@@ -76,23 +89,16 @@
 	@Autowired
 	MosipRouter router;
 
-	@Autowired
```
Comment (loganathan-sekaran) [inline registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidatorStage.java:79]: Do we need to remove this ?

File: registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java [hunk 4]
```diff
@@ -129,10 +140,6 @@
 	@Autowired
 	private RegistrationStatusService<String, InternalRegistrationStatusDto, RegistrationStatusDto> registrationStatusService;
 
-	/** The utilities. */
-	@Autowired
```
Comment (loganathan-sekaran) [inline registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java:133]: Do we need to remove this ?

File: registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java [hunk 5]
```diff
@@ -152,10 +159,6 @@
 	@Autowired
 	private PriorityBasedPacketManagerService packetManagerService;
 
-	/** The cbeff util. */
-	@Autowired
```
Comment (loganathan-sekaran) [inline registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java:156]: Do we need to remove this ?

File: registration-processor/post-processor/registration-processor-printing-stage/pom.xml (hunk 1 above)
Comment (manojsp12) [reply registration-processor/post-processor/registration-processor-printing-stage/pom.xml:112]: Without this dependency, application is throwing NoClassFoundException: io.vertx.core.Promise

File: registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/config/OSIConfigBean.java (hunk 2 above)
Comment (manojsp12) [reply registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/config/OSIConfigBean.java:26]: These beans are added as part of componentScan in OSIValidatorStage

File: registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidatorStage.java (hunk 3 above)
Comment (manojsp12) [reply registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidatorStage.java:79]: These beans are unused in OSIValidatorStage

File: registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java (hunk 4 above)
Comment (manojsp12) [reply registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java:133]: These beans are unused in QualityCheckerStage

File: registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java (hunk 5 above)
Comment (manojsp12) [reply registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java:156]: These beans are unused in QualityCheckerStage

**Review by loganathan-sekaran — APPROVED** [review]
Approved.

**Pull request merged**